addons/spreadsheet_dashboard/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Spreadsheet dashboard",
     "version": "1.0",
-    "category": "Hidden",
+    "category": "Productivity/dashboard",
```

Comment:
Don't we usually capitalize the first letter of the sub-category? So "Productivity/Dashboard"?